core/storage/topic/gae_models.py

```diff
@@ -113,6 +113,9 @@ class TopicModel(base_models.VersionedModel):
     thumbnail_filename = datastore_services.StringProperty(indexed=True)
     # The thumbnail background color of the topic.
     thumbnail_bg_color = datastore_services.StringProperty(indexed=True)
+    # The thumbnail size in bytes of the topic.
+    thumbnail_size_in_bytes = (
+        datastore_services.IntegerProperty(indexed=True, default=None))
```

Comment:
Why default=None? (I think that happens automatically.)

Reply:
Agreed, updated accordingly.